book/guide/initializing.md

```diff
@@ -120,12 +120,6 @@ network monitoring tools like `tcpdump`.
 
 :::
 
-The BPF program is loaded into `/sys/fs/bpf/<name>/` and will remain
```

Comment:
This `xdp` section needs to be moved since it's no longer part of initializing (`fdctl configure`). The two paragraphs below about `This stage must be run` are no longer relevant either.

Reply:
@mmcgee-jump Can you push a commit to this branch with these changes?